state/applicationoffers_test.go

```diff
@@ -500,16 +518,102 @@ func (s *applicationOffersSuite) addOfferConnection(c *gc.C, offerUUID string) {
 		Username:        "admin",
 		SourceModelUUID: testing.ModelTag.Id(),
 	})
+
+	return app
 }
 
-func (s *applicationOffersSuite) TestRemoveOffersWithConnections(c *gc.C) {
+func (s *applicationOffersSuite) TestRemoveOffersSucceedsWithZeroConnections(c *gc.C) {
+	s.createDefaultOffer(c)
+	ao := state.NewApplicationOffers(s.State)
+	err := ao.Remove("hosted-mysql", false)
+	c.Assert(err, jc.ErrorIsNil)
+	err = s.mysql.Refresh()
```

Comment:
you're not checking that the offer is removed!

Reply:
Opps. I think this one got lost in a rebase.